definition_of_done.md

```diff
@@ -0,0 +1,38 @@
+ Definition of done
+
+This document clarifies, when an product backlog item is ready. During scrum development process this gives us a checklist,
+what needs to be done.
+The individual points are a basis for acceptance criteria and are shortened or appended by specific measures as needed.
+
+## Documentation
+* [ ] Public api (diagrams?)
+* [ ] Tutorials
+* [ ] Readme
+* [ ] ROS wiki
+* [ ] Code (sphinx/doxygen)
+* [ ] Good commit messages ([example](https://dev.to/jacobherrington/how-to-write-useful-commit-messages-my-commit-message-template-20n9
+))
+* [ ] Changelog.md updated
+* [ ] Copyright headers
+
+## Reviews
+* [ ] Architecture
+* [ ] Code (coding rules, style guide)
+* [ ] Test review (test plan and individual test cases)
+* [ ] Documentation review
+
+## Quality: CI pass
+* [ ] Catkin_lint
+* [ ] Coverage
```

Comment:
In my opinion, it should be clear, which coverage measures we take on and how "good" it should be. E.g.
```suggestion
* [ ] Coverage (Line coverage: 100%)
```
This would correspond to our coverage scripts on `pilz_robots`.

Reply:
Or link to [travis](https://github.com/PilzDE/pilz_robots/blob/melodic-devel/.travis.yml)? The CI checks are defined in the script there.